src/display/mod.rs

```diff
@@ -78,7 +78,11 @@ impl Dot for Relation {
         Command::new("open")
             .arg(format!("/tmp/{name}.html"))
             .output()
-            .expect("Error: this works on MacOS");
+            .or(
+                Command::new("xdg-open")
+                .arg(format!("http://localhost:8000/{name}.html"))
```

Comment:
This will not work, remove

Reply:
This is needed to visualise the relations and run some tests individually when the repository is in a remote linux machine. it requires to run a html server in the /tmp/ directory. It is not general but it allows me to run some individual tests in my development environment.